contrib/pc-sanity/units/pc-sanity/pc-sanity.pxu

```diff
@@ -81,7 +81,27 @@ exclude:
     com.canonical.certification::after-suspend-bluetooth4/beacon_.*
 nested_part:
     com.canonical.certification::pc-sanity-base
-    com.canonical.certification::client-cert-auto
+    com.canonical.certification::client-cert-desktop-22-04-automated
+    com.canonical.certification::power-management-cpu-cstate-cert-automated
+bootstrap_include:
+    device
+    graphics_card
+    cpuinfo
+
+id: pc-sanity-smoke-test-24-04
```

Comment:
I think we can remove no gpu switching plan and make no dgpu switching become default. What do you think?

Reply:
yes, I think we can make no dgpu switching become default